### PR TITLE
домашка3

### PR DESCRIPTION
**Pull request by Polina-Bril**
Александр, Роман не подтвердил, отправляю только вам на проверку

### CODE REVIEW

File: FreeIT/src/lesson3/Task1.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package lesson3;
+
+//1)Начав тренировки, спортсмен в первый день пробежал 10 км. Каждыйдень он увеличивал дневную норму на 10% нормы 
+//предыдущего дня. Какойсуммарный путь пробежит спортсмен за 7 дней?
+
+public class Task1 {
+
+	public static void main(String[] args) {
+		float dayProbeg = 10f;
```
Comment (TheFallen058) [inline FreeIT/src/lesson3/Task1.java:9]: Try not to use russian words. dayDistance will be better

File: FreeIT/src/lesson3/Task1.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		float probeg = dayProbeg;
+		for (int i = 2; i <= 7; i++) {
```
Comment (TheFallen058) [inline FreeIT/src/lesson3/Task1.java:11]: You can start counting from 1. in this case cycle will look for (int i = 1; i <7; i++)

File: FreeIT/src/lesson3/Task12book.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package lesson3;
+
+public class Task12book {
+
+//	Найдите сумму первых n целых чисел, которые делятся на 3.
+
+	public static void main(String[] args) {
```
Comment (TheFallen058) [inline FreeIT/src/lesson3/Task12book.java:7]: Something wrong with the logic. We should define n and sum numbers starting from 1 if the is divisible by 3.
ex: n =1 => result 3, n=3 => result 18

File: FreeIT/src/lesson3/Task17book.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
+package lesson3;
+
+public class Task17book {
+
+//	Создать двухмерный квадратный массив, и заполнить его «бабочкой»,
+
```
Comment (TheFallen058) [inline FreeIT/src/lesson3/Task17book.java:6]: You code works as expected.
My solution will be something like this:
for (int i = 0; i < array.length; i++) {
for (int j = 0; j < array.length; j++) {
if (i < array.length / 2) {
if (i <= j && array.length - i > j) {
array[i][j] = 1;
}
} else {
if (i >= j && array.length - i < j + 2) {
array[i][j] = 1;
}
}
}
}
please try to analyze. it uses twice less for circles.

**Review by TheFallen058 — CHANGES_REQUESTED** [review]
Good work. 
Please fix task 12.
Good logic and algorithms, good class naming. In some cases please pay more attention to variable naming.

File: FreeIT/src/lesson3/Task12book.java (hunk 3 above)
Comment (Polina-Bril) [reply FreeIT/src/lesson3/Task12book.java:7]: What's wrong? I don't understand. I use the same logic, as you wrote.

File: FreeIT/src/lesson3/Task1.java (hunk 1 above)
Comment (Polina-Bril) [reply FreeIT/src/lesson3/Task1.java:9]: Ok

File: FreeIT/src/lesson3/Task1.java (hunk 2 above)
Comment (Polina-Bril) [reply FreeIT/src/lesson3/Task1.java:11]: OK. But why its better?

File: FreeIT/src/lesson3/Task17book.java (hunk 4 above)
Comment (Polina-Bril) [reply FreeIT/src/lesson3/Task17book.java:6]: OK, thank you

File: FreeIT/src/lesson3/Task1.java (hunk 2 above)
Comment (TheFallen058) [reply FreeIT/src/lesson3/Task1.java:11]: it shows clear logic. if we look at your code (without reading description) there could be thoughts about starting from day 2 and skipping day one at all. It's days, something that we can count.

File: FreeIT/src/lesson3/Task12book.java (hunk 3 above)
Comment (TheFallen058) [reply FreeIT/src/lesson3/Task12book.java:7]: Try to set n=3 in your code. Result should be 18.  (3+6+9)
for n=4 result should be 30 (3+6+9+12)

File: FreeIT/src/lesson3/Task1.java (hunk 2 above)
Comment (TheFallen058) [reply FreeIT/src/lesson3/Task1.java:11]: Sorry, last night was poor in words.
We should write the code not just for us but for other developers too, someone will support it.
So lets imagine situation that other developer should change the code to support new requirements:
//1)Начав тренировки, спортсмен в первый день пробежал 15 км. Каждый день он увеличивал дневную норму на 10% нормы 
//предыдущего дня. Какой суммарный путь пробежит спортсмен за 10 дней?
Code like this:
		double distancePerDay = 10;
		double totalDistance = 0;
		double improvementPerDay = 1.1;
		for (int days = 1; days <= 7; days++) {
			totalDistance += distancePerDay;
			distancePerDay *= improvementPerDay;
		}
will be better for understanding without looking for task with previous requirements. Everything is clear without additional comments and ready for changes

**Pull request merged**